recipes-rubygems/rubygems-ffi-yajl_2.4.0.bb

```diff
@@ -6,19 +6,19 @@ HOMEPAGE = "http://github.com/chef/ffi-yajl"
 LICENSE = "MIT"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=84928ddb161eab99f094cbe0c187e855"
 
-EXTRA_DEPENDS += "\
+EXTRA_DEPENDS_append += " \
     rubygems-libyajl2 \
     yajl \
 "
-EXTRA_RDEPENDS += "\
```

Comment:
potential issue: Use '_append' instead of ' += ' as it overwrites 'EXTRA_DEPENDS ?= ""